Makefile

```diff
@@ -70,7 +70,7 @@ FTESTS ?= tests/functional
 TESTOPTS ?= -v
 .PHONY: test
 test: ## Run the application tests in parallel (for rapid development)
-	@TEST_CMD="python -m pytest -v -n 4 --ignore=$(FTESTS) --ignore=$(ITESTS) --cov-config .coveragerc --cov-report html --cov-report term-missing --cov=securedrop_client --cov-fail-under 100 $(TESTOPTS) $(TESTS)" ; \
+	@TEST_CMD="python -m pytest -v -n 8 --ignore=$(FTESTS) --ignore=$(ITESTS) --cov-config .coveragerc --cov-report html --cov-report term-missing --cov=securedrop_client --cov-fail-under 100 $(TESTOPTS) $(TESTS)" ; \
```

Comment:
with as many tests as we have, n=8 makes more sense here

Reply:
actually, nvm, 4 still seems optimal for our test environment plus number of tests